software/flyimg.yml

```diff
@@ -0,0 +1,12 @@
+name: "Flyimg"
+website_url: "https://flyimg.io"
+source_code_url: "https://github.com/flyimg/flyimg"
+description: "Dockerized application to resize and crop images on the fly. Get optimised images with MozJPEG, WebP or PNG using ImageMagick, with an efficient caching system."
+licenses:
+  - MIT
+platforms:
+  - Docker
+tags:
+  - Miscellaneous
+depends_3rdparty: no
+demo_url: "https://demo.flyimg.io"
```

Comment:
>  Demo links should only be used for interactive demos, i.e. not video demonstrations.
```suggestion
```

Reply:
actually the demo link is meant to be used in interactive mode, the logo served is generated using the flying API call, and the user can copy the parameters shown in the description and use the demo link.

Reply:
@KieranRobson any updates here?